vivarium/core/serialize.py

```diff
@@ -1,53 +1,19 @@
-from collections.abc import Sequence
 import math
-from typing import Any, Callable, List, Union
 import warnings
+from typing import Any, List, Union
+from collections.abc import Callable
 
 import numpy as np
-from bson import ObjectId
 from pint import Unit
 from pint.quantity import Quantity
+from bson.codec_options import TypeEncoder, TypeRegistry, CodecOptions
 
-from vivarium.core.registry import serializer_registry, Serializer
-from vivarium.core.composer import Composer, Composite
-from vivarium.core.process import Process
-from vivarium.library.dict_utils import remove_multi_update
 from vivarium.library.units import units
+from vivarium.core.registry import serializer_registry, Serializer
 
-
-# Serialization and deserialization functions that handle arbitrary data
-# types.
-
-
-def serialize_value(value: Any) -> Any:
```

Comment:
I think the `serialize_value` interface is worth keeping to avoid breaking the API. It can just call the BSON serialization functions

Reply:
Addressed in 5394975.